docs/py_docs.rst

```diff
@@ -226,7 +223,31 @@ Class/method/function docstrings must be placed directly below the declaration,
        """
        pass
 
-Note that the class docstring takes the place of a docstring for the ``__init__`` method; ``__init__`` has no docstring.
+The class docstring takes the place of a docstring for the ``__init__`` method.
+``__init__`` methods don't have docstrings.
+
+.. _py-docstring-attributes:
+
+Module Constants and Class Attributes
+-------------------------------------
+
+Docstrings for module-level variables and class attributes appear directly below their first declaration.
+For example:
```

Comment:
You might want to link to or mention the policy on attributes defined by `__init__`, since that confused me when I was reading this.